api/specs/web-server/_users.py

```diff
@@ -7,6 +7,7 @@
 from typing import Annotated
 
 from fastapi import APIRouter, Depends, status
+from models_library.api_schemas_webserver.users import ProfileGet, ProfileUpdate
```

Comment:
since you change the entrypoint, would it not be more consistent to also rename ProfileUpdate to ProfilePatch?

Reply:
will follow up in https://github.com/ITISFoundation/osparc-simcore/issues/6913